README.md

```diff
@@ -1670,6 +1671,13 @@ require'nvim_lsp'.metals.setup{}
     settings = {}
 ```
 
+## nimls
+
+https://github.com/PMunch/nimlsp
+
+Nim language server with rich IDE features.
+It can be installed in Nvim with `:LspInstall nimls` or `nimble install nimlsp`
```

Comment:
README.md is automaticaly update by Github Actions.

Reply:
Resolved in 9a5d9b3.